src/data/mv_polynomial/basic.lean

```diff
@@ -818,15 +818,17 @@ section aeval
 
 /-! ### The algebra of multivariate polynomials -/
 
-variables {R : Type u} {A : Type v} {S : Type w} (f : σ → A)
-variables [comm_semiring R] [comm_semiring A] [algebra R A] [comm_semiring S]
+variables {A : Type v} {S : Type w} (f : σ → A)
+variables [comm_semiring A] [algebra R A] [comm_semiring S]
```

Comment:
Reason for the above breakage is this change. As `R` was already being used as a variable in this part of the file, when I changed `α` to `R`, Lean complains that `R` already exists, so I deleted the `R` from this sections.

Reply:
Ok... It would be good to keep the order of variables as much the same as possible. (Otherwise this refactor might ramify into many other files, which seems undesirable.)

Reply:
Actually the build succeeded, so there was only one breakage. I think if I rename `β`, `γ`, `δ` etc. then I can get rid of the `A` on L821 and the type arguments should be together, and the typeclass ones together. The only change would then be `σ S` being changed to `S σ` in the previous comment.

Reply:
Wait, the previous comment isn't right. `comm_semiring R` will still come before `f`.

The easiest way forward is just to use the original `variables` but with new names.

Reply:
I'll let you do what you think is best (-;

Reply:
Alright thanks. If there's only one breakage in mathlib then I'm not too concerned about it to be perfectly honest.